arm-automation/2015-10-31/swagger/webhook.json

```diff
@@ -0,0 +1,363 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The automation account name."
+          },
+          {
+            "name": "webhookName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The webhook name."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/Webhook"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Webhooks"
+        ],
+        "operationId": "Webhooks_CreateOrUpdate",
+        "description": "Create the webhook identified by webhook name.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/webhookoperations"
+        },
+        "x-ms-examples": {
+          "Create or update webhook": {
+            "$ref": "../examples/webhook/createOrUpdateWebhook.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "automationAccountName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The automation account name."
+          },
+          {
+            "name": "webhookName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The webhook name."
+          },
+          {
+            "name": "webhookCreateOrUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./definitions.json#/definitions/WebhookCreateOrUpdateParameters"
+            },
+            "description": "The create or update parameters for webhook."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/Webhook"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/Webhook"
```

Comment:
201 means accepted status. It is not a valid termination of the whole process. So, how is it returning a Webhook?